packages/labs/analyzer/src/test/server/utils.ts

```diff
@@ -244,6 +244,30 @@ export const setupAnalyzerForTest = (
   }
 };
 
+export const setupAnalyzerForNodeTest = (lang: Language, pkg: string) => {
```

Comment:
Can write setupAnalyzerForTest in terms of setupAnalyzerForNodeTest I think:

```ts
  try {
    Object.assign(ctx, setupAnalyzerForNodeTest(lang, pkg))
  } catch (error) {
    // Uvu has a bug where it silently ignores failures in before and after,
    // see https://github.com/lukeed/uvu/issues/191.
    console.error('uvu before error', error);
    process.exit(1);
  }
}

Reply:
nice, done!